README.md

```diff
@@ -1,2 +1,5 @@
 # Demo
 Demonstration Repository for Team Members
+
```

Comment:
add two blank lines

Reply:
lines added to readme per review comments